loadlsststack.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+LSSTSW_DIR="$(dirname "$(pwd)")/lsstsw"
+
+GCP_BUCKET="gs://eups-lsstsw-cache"
+
+TMP_FOLDER="/tmp/lsstswcache"
+
+fail() {
+  echo "$1" >&2
+  exit 1
+}
+
+[[ $# -eq 0 ]] && fail "No tag was passed" 
+# shellcheck disable=SC2054 disable=SC2034
+OPTS=$(getopt -o '' -l tag: -- "$@")
+eval set -- "$OPTS"
+while true
+do
+  case "$1" in
+    --tag)             DATE_TAG=$2             ; shift 2 ;;
+    --) shift; break;;
+    *) [[ "$*" != "" ]] && fail "Unknown option: $1"
+       break;;
+  esac
+done
+
+ARCH_VER=$(uname -m)
+
+TARGET="${DATE_TAG}_${ARCH_VER}_lsstsw.tar.gz" 
+mkdir -p "${TMP_FOLDER}"
+
+
+cd "$LSSTSW_DIR" || fail "Can't find ${LSSTSW_DIR}"
```

Comment:
Should this already exist, if it's a fresh instance?  Maybe `mkdir -p` here before the `cd`.